docs/tutorials/using-host-labels.md

```diff
@@ -0,0 +1,204 @@
+
+```bash
+cd /etc/netdata   # Replace this path with your Netdata config directory, if different
+sudo ./edit-config netdata.conf
+```
+
+Create a new `[host labels]` section wherever you choose:
+
+```conf
+[host labels]
+```
+
+Beneath that section, you can define a new host label, and its value for the system in question, taking care not to
+violate any of the [host label naming rules](../configuration-guide.md#netdata-labels), like so:
+
+```conf
+[host labels]
+    type = webserver
+    location = us-seattle
+    installed = 20200218
+```
+
+Once you've written a few host labels, you need to enable them. Instead of restarting the entire Netdata service, you
+can reload labels using the helpful `netdatacli` tool:
+
+```bash
+netdatacli reload-labels
+```
+
+Your host labels will now be enabled. You can double-check these by using `curl http://HOST-IP:19999/api/v1/info` to
+read the status of your agent. For example, from a VPS system running Debian 10:
+
+```json
+{
+  ...
+  "host_labels": {
+    "_is_master": "false",
+    "_virt_detection": "systemd-detect-virt",
+    "_container_detection": "none",
+    "_container": "unknown",
+    "_virtualization": "kvm",
+    "_architecture": "x86_64",
+    "_kernel_version": "4.19.0-6-amd64",
+    "_os_version": "10 (buster)",
+    "_os_name": "Debian GNU/Linux",
+    "type": "webserver",
+    "location": "seattle",
+    "installed": "20200218"
+  },
+  ...
+}
+```
+
+You may have noticed a handful of labels that begin with an underscore (`_`). These are automatic labels.
+
+### Automatic labels
+
+When Netdata starts, it captures relevant information about the system and converts them into automatically-generated
+host labels. You can use these to logically organize your systems via health entities, exporting metrics,
+streaming/master status, and more.
+
+They capture the following:
+
+-   Kernel version
+-   Operating system name and version
+-   CPU architecture, system cores, CPU frequency, RAM, and disk space
+-   If Netdata is running inside of a container
+    -   If so, OS and hardware details about the container's host
+-   What virtualization layer the system runs on top of, if any
+-   Whether the system is a streaming master or slave
+
+If you want to organize your systems without manually creating host tags, try the automatic labels in some of the
+features below.
+
+## Host labels in streaming
+
+You may have noticed the `_is_master` and `_is_slave` automatic labels from above. Host labels are also now streamed
+from a slave to its master agent, which concentrates an entire infrastructure's OS, hardware, container, and
+virtualization information in one place: the master.
+
+Now, if you'd like to remind yourself of how much RAM a certain slave system has, you can simply access
+`http://localhost:1999/host/SLAVE_NAME/api/v1/info` and reference the automatically-generated host labels from the slave
```

Comment:
You missed a `9` to the default port that is `19999`, case you decide to keep the port as `1999` I think it would be good to say for our users that in this example, you are not using default values.

Reply:
Fixed to `19999`!